app/Factories/UserFactory.php

```diff
@@ -6,7 +6,7 @@
 use App\Helpers\CryptoHelper;
 
 class UserFactory {
-    public static function createUser($username, $email, $password, $active=0, $ip='127.0.0.1', $api_key=false, $api_active=0, $is_admin = false) {
+    public static function createUser($username, $email, $password, $api_key, $active='0', $api_active=FALSE, $ip='127.0.0.1', $is_admin = FALSE) {
```

Comment:
We should retain the old format. As far as I know, there are also some tests depending on the original order of the arguments. We should refrain from making unnecessary changes if possible :)

i.e, $ip, $api_key, ....
instead of $api_key, $active, $api_active, $ip...